README.md

```diff
@@ -62,3 +65,29 @@ fn example(bytes: &mut Bytes) {
         .unwrap();
 }
 ```
+
+## Serde
+*Requires `serde` feature.*
+
+```ignore
```

Comment:
I don't like this, because it breaks highligthing but haven't found any solution

Reply:
What do you think about adding `doctest = false`?

Reply:
If you can disabled just this one doctest while keeping the ```rust for proper highlighting then sure.

Reply:
It disables testing all doctests

Reply:
Wohoo, I managed to make it compile after your last force push

```diff
diff --git a/README.md b/README.md
index 76535c7..a7f3634 100644
--- a/README.md
+++ b/README.md
@@ -69,11 +69,11 @@ fn example(bytes: &mut Bytes) {
 ## Serde
 *Requires `serde` feature.*
 
-```ignore
-use crab_nbt::serde::arrays::IntArray;
+```rust
+use crab_nbt::serde::{arrays::IntArray, ser::to_bytes_unnamed, de::from_bytes_unnamed};
 use serde::{Deserialize, Serialize};
 
-#[derive(Deserialize, PartialEq, Debug)]
+#[derive(Serialize, Deserialize, PartialEq, Debug)]
 struct Test {
     number: i32,
     #[serde(with = "IntArray")]
@@ -85,9 +85,9 @@ fn cycle() {
         number: 5,
         int_array: vec![7, 8]
     };
-    let bytes = to_bytes_unnamed(&test).unwrap();
-    let recreated_struct: Test = from_bytes_unnamed(&mut expected).unwrap();
+    let mut bytes = to_bytes_unnamed(&test).unwrap();
+    let recreated_struct: Test = from_bytes_unnamed(&mut bytes).unwrap();
     
     assert_eq!(test, recreated_struct);
 }
-```
\ No newline at end of file
+```
```